mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -119,3 +119,11 @@ The provider will now enforce at plan time that at most one of these fields be s
 ### `deletion_policy` now defaults to `DELETE`
 
 Previously, `google_firebase_web_app` deletions default to `ABANDON`, which means to only stop tracking the WebApp in Terraform. The actual app is not deleted from the Firebase project. If you are relying on this behavior, set `deletion_policy` to `ABANDON` explicitly in the new version.
+
+## Resource: `google_sql_database`
+
+### `database_flags` is now a set
+
+Previously, `database_flags` was a list, making it order-dependent. It is now a set.
+
+If you were relying on accessing an individual flag by index (for example, `google_sql_database_instance.instance.settings.0.database_flags.0.name`), then that will now need to by hash (for example, `google_sql_database_instance.instance.settings.0.database_flags.<some-hash>.name`).
```

Comment:
Example copied from @megan07's comment at https://github.com/GoogleCloudPlatform/magic-modules/pull/8118#issuecomment-1589548637.

As they said, this is very unlikely, but I guess it doesn't hurt to mention it.

Reply:
Agreed, that is worth adding. 

Would you mind adding the upgrade guide change in a separate PR in the `main` branch instead of in this PR? That gets published and allows users to preview the changes and required steps ahead of time.